docs/core.rst

```diff
@@ -408,39 +408,48 @@ Calling :py:func:`~orca.orca.run` with just a list of steps,
 as in the above example, will run through the steps once.
 To run the pipeline over some a sequence, provide those values as a sequence
 to :py:func:`~orca.orca.run` using the ``iter_vars`` argument.
-The variable ``iter_var`` is provided as an injectable to Orca functions:
+The variables ``iter_var`` and ``iter_step`` are provided as injectables to Orca functions.
+The ``iter_var`` injectable stores the year of the current simulation iteration, and the 
+``iter_step`` injectable is a tuple that stores the sequence position and name of the current
+simulation step in the iteration:
 
 .. code-block:: python
 
     In [77]: @orca.step()
-       ....: def print_year(iter_var):
+       ....: def print_year(iter_var,iter_step):
        ....:         print '*** the iteration value is {} ***'.format(iter_var)
+       ....:         print '*** step {0} is {1} ***'.format(iter_step[0],iter_step[1])
```

Comment:
Use the namedtuple fields here instead of position, it'll be more readable:

```
print '*** step number {0} is named {1} ***'.format(iter_step.step_num, iter_step.step_name)
```